components/charts/__tests__/__snapshots__/AdvancedTimeline-test.tsx.snap

```diff
@@ -187,11 +187,13 @@ exports[`should render correctly 1`] = `
         />
       </g>
     </g>
-    <g />
+    <g
+      transform="translate(0, 20)"
+    />
     <g>
       <path
         className="line-chart-path line-chart-path-0"
-        d="MNaN,40LNaN,20"
+        d="MNaN,26.666666666666668LNaN,13.333333333333336"
```

Comment:
There are some `NaN`s here, is it expected ?

Reply:
No, but they were already there, and I have no idea why. And didn't feel like debugging this 🤷‍♂ .

Reply:
Mind you, this comes from d3. Hence, me not being motivated to understand why it doesn't work in our test env 😅 .

Reply:
@siegfried-ehret-sonarsource Is this blocking for you, or do you approve the changes?

Reply:
Approved ! We already haved some weird values before, let's continue :).
This will be interesting to investigate.